src/KubernetesClient/generated/IKubernetes.Watch.cs

```diff
@@ -7,7 +7,8 @@
 using System; 
 using System.Collections.Generic; 
 using System.Threading; 
-using System.Threading.Tasks; 
+using System.Threading.Tasks;
```

Comment:
generated code should be modified by code generator only

Reply:
> not quite get the idea of this PR?
> seems nothing changed but some code format?
Yes, more or less. This is just an initial PR to take a look at the code, clean up some issues I found while reading along the way, and get a first PR into the repo. I wasn't aware that some of this was code generated, so I'll take a look further and go from there. 😄

Reply:
Ah, I see now the error. This is generated code. Closing the pr...